bears/r/FormatRBear.py

```diff
@@ -22,8 +26,15 @@ class FormatRBear:
     LANGUAGES = {'R'}
     AUTHORS = {'The coala developers'}
     AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
-    REQUIREMENTS = {RscriptRequirement(package='formatR', flag='-e',
-                                       repo='http://cran.rstudio.com')}
+    REQUIREMENTS = {
+        AnyOneOfRequirements(
+            [DistributionRequirement(
+                apt_get='r-cran-formatr',
+                zypper='R-formatR',
+             ),
+             RscriptRequirement(package='formatR'), ]
```

Comment:
put the closing bracket on a new line, with its own closing bracket, so these lines dont need to be modified.